source/ios/AdaptiveCards/AdaptiveCards/AdaptiveCards/ACOBaseActionElement.h

```diff
@@ -9,6 +9,18 @@
 
 @interface ACOBaseActionElement:NSObject
 
+typedef NS_ENUM(NSInteger, ACRActionType) {
+    ACRShowCard = 1,
+    ACRSubmit,
+    ACROpenUrl,
+};
+
+@property ACRActionType type;
+
+- (NSString *)title;
+- (NSString *)id;
```

Comment:
>id [](start = 14, length = 2)

I wouldn't recommend calling this method `id`. It's not illegal, but that identifier already has a meaning in Objective-C.

Reply:
thanks


---
In reply to: [172390942](https://github.com/Microsoft/AdaptiveCards/pull/1211#discussion_r172390942) [](ancestors = 172390942)